starter/main.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"context"
+	"log"
+	"time"
+
+	"github.com/google/uuid"
+	"go.temporal.io/sdk/client"
+
+	subs "github.com/afitz0/temporal-subscriptions-go"
+	"github.com/afitz0/temporal-subscriptions-go/zapadapter"
+)
+
+func main() {
+	c, err := client.Dial(client.Options{
+		Logger: zapadapter.NewZapAdapter(
+			zapadapter.NewZapLogger()),
+	})
+	if err != nil {
+		log.Fatalln("Unable to create client", err)
+	}
+	defer c.Close()
+
+	uid := uuid.New()
+
+	customer := subs.CustomerInfo{
+		Name:  "Fitz",
+		Email: "a@a.com",
+		UID:   uid.String(),
+	}
+
+	workflowOptions := client.StartWorkflowOptions{
+		ID:        "cid-" + customer.UID,
+		TaskQueue: "subscriptions",
+	}
+
+	// SUPER short periods to allow for easier testing.
+	sub := subs.SubscriptionInfo{
+		TrialPeriodExpiration: time.Now().Add(time.Duration(time.Second * 5)),
```

Comment:
```suggestion
		TrialPeriodExpiration: time.Now().UTC().Add(5 * time.Second),
```

Not a big deal since timezone serialized on the time string, but usually better for clarity and readability of JSON. When multiplying by a const by duration the result is automatically a duration.